portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// use modern JavaScript (ES5)
+"use strict"
+
 /**
- * Adds a random greeting to the page.
+ * Adds a cycling greeting to the page.
  */
+let greetingIndex = 0;
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!', 'Hallo Welt!'];
+
+  // Pick the next greeting in a cycle.
+  const greeting = greetings[greetingIndex];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Update greeting index.
```

Comment:
Remove the whitespace before this line

Reply:
Are you referring to removing the empty line at line 28? Thanks!

Reply:
Yep